scripts/clang.jl

```diff
@@ -0,0 +1,74 @@
+# TODO(odow):
+#
+# This script can be used to build the C interface to Gurobi. However, it requires
+# you to manually do the following steps first:
+#
+# 1) Copy cplex.h from CPLEX into this /scripts directory
+# 2) Copy cpxconst.h from CPLEX into this /scripts directory
+
```

Comment:
an alternative followed in HiGHS consists in letting the user specify the CPLEX location to get the headers:
https://github.com/jump-dev/HiGHS.jl/blob/master/gen/gen.jl#L3

Reply:
I think it's fine as-is. This is only meant to be run very infrequently. I found it very useful having the header file when developing.